GitUI/UserControls/RevisionGrid/RevisionGridControl.Designer.cs

```diff
@@ -120,6 +123,9 @@ private void InitializeComponent()
             this.stopBisectToolStripMenuItem,
             this.bisectSeparator,
             this.copyToClipboardToolStripMenuItem,
+            this.applyStashToolStripMenuItem,
```

Comment:
Please add a separator after the "Copy to clipboard" menu item.

Reply:
Changed this in the latest commit before squashing and updating the commit message.